<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RayTracerClass</class>
 <widget class="QMainWindow" name="RayTracerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>547</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RayTracer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="OutputWindow" name="mainWindow" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Sample</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNumSample">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>D0; </string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxThread">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectColor"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSelectColor">
   <property name="text">
    <string>SelectColor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OutputWindow</class>
   <extends>QWidget</extends>
   <header>outputwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="raytracer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>RayTracerClass</receiver>
   <slot>OnClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectColor</sender>
   <signal>triggered()</signal>
   <receiver>RayTracerClass</receiver>
   <slot>OnSelectColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnClear()</slot>
  <slot>OnSelectColor()</slot>
 </slots>
</ui>
